lambda-src/lambda_function.py

```diff
@@ -0,0 +1,23 @@
+import json
+import urllib.parse
+import boto3
+
+print('Loading function')
```

Comment:
do you need the print?

Reply:
> the part with dynamo db I already commented 🤔
I have changed it in a earlier branch which is not merged already because I am waiting for your answer. How is a good practice in this case? change it in all later branches?

The print is only for testing.

Reply:
> the part with dynamo db I already commented 🤔
I am not sure what do you mean...